### PR TITLE
kommander 0.4.31

### PR DESCRIPTION
**Pull request by jr0d**
Manually adds kommander-0.4.31 //

Change to chart: 

```diff
4d3
< engine: gotpl
18d16
< tillerVersion: '>=2.12.0'
diff -r 30/kommander/charts/kubeaddons-catalog/Chart.yaml 31/kommander/charts/kubeaddons-catalog/Chart.yaml
2c2
< appVersion: v0.8.2
---
> appVersion: v0.8.3
8c8,9
< - name: alejandroEsc
---
> - email: aescobar@d2iq.com
>   name: alejandroEsc
12c13
< version: 0.1.6
---
> version: 0.1.11
diff -r 30/kommander/charts/kubeaddons-catalog/files/crd_addonrepository.yaml 31/kommander/charts/kubeaddons-catalog/files/crd_addonrepository.yaml
1d0
< ---
5,6d3
<   creationTimestamp: null
<   name: addonrepositories.kubeaddons.mesosphere.io
7a5
>     controller-gen.kubebuilder.io/version: v0.2.5
8a7,8
>   creationTimestamp: null
>   name: addonrepositories.kubeaddons.mesosphere.io
20c20,21
<   scope: ""
---
>   preserveUnknownFields: false
>   scope: Namespaced
30c31
<             internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
---
>             internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
35c36
<             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
---
>             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
59c60,64
<               type: string
---
>               anyOf:
>                 - type: integer
>                 - type: string
>               pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
>               x-kubernetes-int-or-string: true
82a88,90
>       storage: false
>     - name: v1beta2
>       served: true
86,87c94,95
<     kind: ""
<     plural: ""
---
>     kind: ''
>     plural: ''
89c97
<   storedVersions: []
\ No newline at end of file
---
>   storedVersions: []
diff -r 30/kommander/charts/kubeaddons-catalog/templates/deployment.yaml 31/kommander/charts/kubeaddons-catalog/templates/deployment.yaml
28a29,33
>           env:
>             {{- range $key, $value := .Values.env }}
>             - name: {{ $key | quote }}
>               value: {{ tpl $value $ | quote }}
>             {{- end }}
diff -r 30/kommander/charts/kubeaddons-catalog/values.yaml 31/kommander/charts/kubeaddons-catalog/values.yaml
5c5
<   tag: v0.8.2
---
>   tag: v0.8.6
13c13
<   enable: true
---
>   enable: false
33a34,35
> 
> env: {}
diff -r 30/kommander/Chart.yaml 31/kommander/Chart.yaml
10c10
< version: 0.4.30
---
> version: 0.4.31
diff -r 30/kommander/requirements.lock 31/kommander/requirements.lock
4c4
<   version: 0.1.6
---
>   version: 0.1.11
19,21c19,21
<   version: 4.5.1
< digest: sha256:6dd99439a773e290b87eff4ad888491bb0ad5dce284865f0357e50763a211ec3
< generated: "2020-05-01T10:27:31.579612552+01:00"
---
>   version: 4.5.*
> digest: sha256:f7cc9d374fb4dd367f045be5eea190f901426d1f64ad0f647565511ba79f6272
> generated: "2020-07-15T18:23:46.982990927-05:00"
diff -r 30/kommander/requirements.yaml 31/kommander/requirements.yaml
3c3
<     version: "0.1.6"
---
>     version: "0.1.11"
```

### CODE REVIEW

File: stable/index.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ entries:
   argo-cd:
   - apiVersion: v1
     appVersion: 1.2.0
-    created: "2020-07-15T19:10:59.142218724Z"
+    created: "2020-07-15T18:33:35.674604774-05:00"
```
Comment (samvantran) [inline stable/index.yaml:6]: that's odd, should it have updated all these non-Kommander charts?

Comment (jr0d) [reply stable/index.yaml:6]: helm rebuilds the whole index it seems.

Comment (jr0d) [reply stable/index.yaml:6]: the previous timestamps all are within milliseconds, so it seems like this happens for every build.

Comment (samvantran) [reply stable/index.yaml:6]: yeah guessing it probably happens from CI or every time someone pushes up a new chart

Comment (samvantran) [reply stable/index.yaml:6]: yup, if you look at any of the commits, dispatch bot does the same thing https://github.com/mesosphere/charts/commits/gh-pages

**Pull request merged**